nano/node/election_scheduler.cpp

```diff
@@ -140,6 +140,10 @@ void nano::election_scheduler::run ()
 					election->transition_active ();
```

Comment:
Is there an else clause (with a lock.unlock ()) missing here as well?
If we think we do not need it an else clause then lets add with an assert zero.

Reply:
There is no problem here. I misread the code because the indentation appears wrong in the browser.